docs/examples/webpack/webpack.config.js

```diff
@@ -83,25 +100,29 @@ module.exports = ({ WEBPACK_SERVE }, { mode }) => ({
 
   output: {
     clean: true,
-    filename: 'assets/javascripts/[name].min.js',
+    filename: 'javascripts/[name].min.js',
     iife: true,
     path: destPath,
     publicPath: '/'
   },
 
   plugins: [
+    new MiniCssExtractPlugin({
+      filename: 'stylesheets/[name].min.css'
+    }),
     new CopyPlugin({
       patterns: [
-        join(srcPath, 'index.html'),
-        {
-          context: join(frontendPath, './src/govuk/assets'),
-          from: '{fonts,images}/**',
-          to: 'assets'
-        }
+        join(srcPath, 'index.html')
       ]
     })
   ],
 
+  resolve: {
+    alias: {
+      '/assets/': join(frontendPath, './src/govuk/assets/')
+    }
+  },
```

Comment:
*question*: What's the role of this `alias`?

Reply:
It's a webpack thing so it knows where assets (like fonts) are:

```css
src: url(/assets/fonts/light-94a07e06a1-v2.woff2)
```

It treats CSS as a module, just like JavaScript

Reply:
So you can turn on things like "asset fingerprinting" or removed unused assets from build output

Reply:
Sounds good, cheers for clarifying 😊 If I follow:
- sass-loader compiles the URL of the assets to `/assets/...`, because of our default `$govuk-assets-path`
- css-loader picks up these URLs, but needs to understand what `/assets/` mean so we need the alias 😊 

Is that it?

Reply:
Yep! We then have to extract it into a file, as the default will be CSS-in-JS otherwise